### ISSUE TITLE
change or hide UpStage logo on live stage?

### ISSUE THREAD

**Issue by helenvarley** [upstage-org/upstage#703]
meta theater asked if it's possible to change or hide the UpStage logo in the upper left-hand corner of the live stage.
is this technically possible?
i think it is a good option to offer, as i know with other platforms i have been annoyed by being forced to have branding that becomes part of a performance.
i imagine that this could be a setting in the stage customisation, with the default being the UpStage logo and there being the possibility to upload a different image to go there.
if there was no image at all, there would be no link back to the foyer. maybe this doesn't matter. 

thoughts?

**Comment by vickis** [comment]
I thought we discussed this and decided it was good to have there because
basically it helps others find us....
if the 'user/client/whatever had their own logo or image to replace that
with but it still leads back to a foyer that includes the link to UpStage
community etc then that could be good...

**Comment by helenvarley** [comment]
yes we did discuss it & decide to have it there, but i can imagine that i might want to remove it for a performance myself. 

as it's a link back to the foyer, i just had the idea that maybe a kind of "EXIT" sign like you have for fire exits in the cinema & theatre would be a good alternative? but also that could be dimmed or turned off ... ?

**Comment by elisadcg** [comment]
![Screenshot_20211213_091008](https://user-images.githubusercontent.com/7611913/145775150-ef9d586b-ec1b-49b0-8882-72ae880374fa.png)
ok one of this 2 icons ? black and green to remember the color identity of UpStage ?

**Comment by helenvarley** [comment]
i love the little running person :laughing: but i wonder if it will be clear enough? the other one is also good, & perhaps a bit more clear that it is to exit. green & black colours are great.
should we test it with some others? ask them what they think these icons might mean, without explaining first?

**Comment by vickis** [comment]
Agree runner is fabulous Elisa :D
Think is very recognisable exit icon
Agree good to test also

**Comment by helenvarley** [comment]
could it be that in the customisation section of stage management, we can choose what to have in that top right-hand corner? the default could be the UpStage logo, as it is now, with both of these exit icons as alternatives, and also the option to upload your own logo.
the link would still be back to the foyer (or also customisable?)

**Comment by vickis** [comment]
Yes sounds ideal :)

**Comment by helenvarley** [comment]
feedback from elsa @ meta theater on the proposed images: " I would prefer the right option, the running person looks a bit too much like a sign for an emergency exit 😊 But the right one is subtle but clear enough to signal an exit."

**Comment by vickis** [comment]
Ah yes - it was the emergency exit link that made me prefer the right one
(both describe the function so I am fine with the majority decision :)

**Comment by helenvarley** [comment]
@nguyenhongphat0 is it possible to have several options? the default being the UpStage logo, then both of these icons as alternative exit icons, & also the option to upload your own?

**Comment by gloriajw** [comment]
I personally think you should always keep the UpStage logo. It' Open Source, but it's still your creation.

**Comment by helenvarley** [comment]
i agree; this was a request from someone ages ago so let's close it & forget about it 😎
